### DISCUSSION TITLE
Permission error trying to setup FreshRSS during database selection

### DISCUSSION THREAD

**Discussion by thiagohartmann** [FreshRSS/FreshRSS#5847] [Q&A]
Environment : Synology using Docker 

Docker Compose:

version: '3.3'
services:
  freshrss:
    image: lscr.io/linuxserver/freshrss:latest
    container_name: freshrss
    ports:
      - 4221:80
    volumes:
      - /volume1/docker/freshrss:/config
    environment:
      - PUID= XX (created a dockerlimited account that has access to my synology /docker/ folder)
      - PGID=YY 
      - TZ=Europe/Zurich
    network_mode: synobridge
    restart: on-failure:5

Issue : During initial setup I get all geen checks, bu then on the step to select the database (sqlite) I cannot advance to the next screen. I click and nothing happens.

Checking the logs inside the docker/freshrss folder in Synology, there is this error always when I click on the button to go to the "next step" during DB selection :

2023/11/10 03:47:28 [error] 295#295: *4 FastCGI sent in stderr: "PHP message: PHP Warning:  file_put_contents(/app/www/data/config.php): Failed to open stream: Permission denied in /app/www/app/install.php on line 151" while reading response header from upstream, client: 172.20.0.1, server: _, request: "POST /i/index.php?step=2 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.1.8:4221", referrer: "http://192.168.1.8:4221/i/index.php?step=2"

PUID and PGID are correct, I tried to play around with users, and folder permissions and owner, but no success.

I initially followed the mariushosting guide, but have been trying to debug, delete container, recreate again without success.

Any ideas?

**Comment by Alkarex** [comment]
Hello,
It looks like there are wrong write permissions on your `/volume1/docker/freshrss`.
You can alternatively use a Docker volume so that the rights will be handled automatically.
Note: our official image is [`freshrss/freshrss`](https://hub.docker.com/r/freshrss/freshrss) so the path for the data is different in yours.
See our documentation https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md
